code/index.html

```diff
@@ -5,32 +5,49 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="stylesheet" href="style.css" />
-    <title>Technigo Project Cluedo</title>
+    <title>Cluedo</title>
   </head>
   <body>
+    <header class="header">
+      <div class="headingcontainer">
+        <h1 class="heading">CL<span class="headingu">U</span>EDO</h1>
+      </div>
+    </header>
     <section class="cards">
-      <article class="killer-deck" id="killerCard">
+      <article class="killer-deck"  id="killerCard">
+        <div class="loader-container">
+          <div id="loaderKiller"></div>
+        </div>
         <p>KILLER</p>
         <p id="killerName"></p>
         <p id="killerAge"></p>
         <p id="killerOccupation"></p>
         <img id="killerImage" src="" height="100" />
         <p id="killerDescription"></p>
+        <p id="favouriteWeapon"></p>
       </article>
       <article class="weapon-deck" id="weaponCard">
+        <div class="loader-container">
+          <div id="loaderWeapon"></div>
+        </div>
         <p>WEAPON</p>
         <p id="weaponName"></p>
         <p id="weaponWeight"></p>
+        <img id="weaponImage" src="" height="100" />
       </article>
       <article class="room-deck" id="roomCard">
+        <div class="loader-container">
+          <div id="loaderRoom"></div>
+        </div>
         <p>ROOM</p>
         <p id="roomName"></p>
+        <img id="roomImage" src="" height="100" />
```

Comment:
You spent time on choosing nice pictures, I think you should adjust their height so they get bigger, there is more space on the card. 
Furthermore, I noticed that your images were huge. I think that it is the reason why they are loading so slowly on your web page. Maybe you could use a tool as https://resizeimage.net/ and reduce them?